languages.yaml

```diff
@@ -1887,7 +1920,7 @@ Perl:
       - cyan
     chip: "#0298C3"
   icon: '\u{E67E}'
-Perl6:
+Raku:
```

Comment:
The languages are alphabetized, so this would need to be moved.

Reply:
fixed. Moved to after `Racket`